### DISCUSSION TITLE
WebDriver Chrome shm_size how to install via containerBuilder?

### DISCUSSION THREAD

**Discussion by Zombach** [testcontainers/testcontainers-dotnet#1070] [Q&A] [answered]
Can anyone tell me how to set shm_size via builder container? I'm using selenium chrome

**Comment by HofmeisterAn** [comment, marked as answer]
The container builder API exposes low level configurations through `WithCreateParameterModifier(Action<TCreateResourceEntity>)`. You can use this to set `shm_size`.

    WithCreateParameterModifier(modifier => modifier.HostConfig.ShmSize = 64000000)
    
The unit is bytes.
